### PR TITLE
Quizard dev

### PR DESCRIPTION
**Pull request by maxiggle**
# Added the following
- Basic Flutter demo code
- MIT LICENSE
- Contribution guide

### CODE REVIEW

File: contribution.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+[fork guide]: https://guides.github.com/activities/forking/#fork
+[pubspec doc]: https://dart.dev/tools/pub/pubspec
+[conventional commit]: https://www.conventionalcommits.org
+
+[github issue]: https://github.com/myConsciousness/freezer/issues/new
+[github issues]: https://github.com/myConsciousness/freezer/issues/new
+[prs]: https://github.com/myConsciousness/freezer/pulls
+[fork guide]: https://guides.github.com/activities/forking/#fork
+[pubspec doc]: https://dart.dev/tools/pub/pubspec
+[conventional commit]: https://www.conventionalcommits.org
```
Comment (CeoFred) [inline contribution.md]: Wrong GitHub repo URLs

File: contribution.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+- If you'd like to submit a fix for a bug, please read the [How To](#12-how-to-contribute) for how to
+  send a Pull Request.
+- Indicate on the open issue that you are working on fixing the bug and the issue will be assigned
+  to you.
+- Write `Fixes #xxxx` in your PR text, where xxxx is the issue number (if there is one).
+- Include a test that isolates the bug and verifies that it was fixed.
+
+### 1.1.3. New Features
+
+- If you'd like to add a feature to the library that doesn't already exist, feel free to describe
```
Comment (CeoFred) [inline contribution.md]: This is not a library though

File: contribution.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  the feature in a new [GitHub issue].
+- If you'd like to implement the new feature, please wait for feedback from the project maintainers
+  before spending too much time writing the code. In some cases, enhancements may not align well
+  with the project objectives at the time.
+- Implement the code for the new feature and please read the [How To](#12-how-to-contribute).
+
+### 1.1.4. Documentation & Miscellaneous
+
+- If you have suggestions for improvements to the documentation, tutorial or examples (or something
```
Comment (CeoFred) [inline contribution.md]: I doubt if we would have tutorials for this project or examples.. not a library

File: contribution.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+### 1.2.1. Requirements
+
+For a contribution to be accepted:
+
+- Documentation should always be updated or added.\*
+- Examples should always be updated or added.\*
+- Tests should always be updated or added.\*
+- Start your PR title with a [conventional commit] type
+  (`feat:`, `fix:` etc).
```
Comment (CeoFred) [inline contribution.md]: Your PR does not include this prefix

**Review by CeoFred — CHANGES_REQUESTED** [review]

**Comment by CeoFred** [comment]
I cannot see the basic flutter code, also please... PRs are to be made to staging branch

File: LICENSE [hunk 5]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 QUIZARD
```
Comment (CeoFred) [inline LICENSE]: Year is 2023

File: contribution.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+or PR. You can still continue to add more commits to the branch you have sent the Pull Request from
+and it will be automatically reflected in the PR.
+
+## 1.3. Open an issue and fork the repository
+
+- If it is a bigger change or a new feature, first of all
+  [file a bug or feature report][github issues], so that we can discuss what direction to follow.
+- [Fork the project][fork guide] on GitHub.
+- Clone the forked repository to your local development machine
+  (e.g. `git clone git@github.com:<YOUR_GITHUB_USER>/freezer.git`).
```
Comment (CeoFred) [inline contribution.md]: freezer.git should quizard-mobile.git instead

**Review by CeoFred — CHANGES_REQUESTED** [review]
Why are your files inside a quizard folder? can you fix this directory issue.. also, change your destination branch to pull into staging

**Comment by maxiggle** [comment]
> Why are your files inside a quizard folder? can you fix this directory issue.. also, change your destination branch to pull into staging

That file is where I have the Flutter project

**Pull request closed without merging**